types/postman-collection/index.d.ts

```diff
@@ -174,25 +176,21 @@ export interface CollectionDefinition extends ItemGroupDefinition {
         name?: string;
         version?: string;
     };
-    item?: Array<ItemDefinition | ItemGroupDefinition>;
     variable?: VariableDefinition;
-    auth?: RequestAuthDefinition;
-    event?: EventDefinition[];
-    version?: string | Version;
 }
 
-export class Collection extends ItemGroup<Request> implements CollectionDefinition {
+export class Collection extends ItemGroup<Request> {
```

Comment:
Hi!  I just found a mistake here. It must be ItemGroup<Item> here, not <Request>. Could you please check this? Thank you!

Reply:
i tried to use code like this but tslint told me it must be Request. If I put request - i always get default request values in my item at the end.

```
    let item = new Item({
      request: { url: url.toString(), method: init.method }
    });
    this.items.append(item);
```